csm-1.5.0-cray-hms-rts-CASMTRIAGE-6698/README.md

```diff
@@ -0,0 +1,38 @@
+# CASMTRIAGE-6698 - cray-hms-rts-snmp pod stuck in CLBO
+
+## Prerequisites
+
+- CSM versions 1.5.0
+  - This fix is included in CSM versions 1.5.1 and up
+
+## Changelog
+
+This hotfix contains the following fixes and enhancements.
+
+- CASMTRIAGE-6698 - cray-hms-rts-snmp pod stuck in CLBO
+  - Fixed concurrency issue associated with RedisActivePipeline
+- CASMHMS-6099 - Add TTL setting to cray-hms-rts-init job
+
+## Installation
+
+The `install-hotfix.sh` script may be run on any Master or Worker Non-Compute Node.
+
+Example:
+
+```bash
+./install-hotfix.sh
+```
+
+## Rollback
+
+To revert `cray-hms-rts` to the previous version, select revision to rolback to:
+
+```bash
+helm -n services history cray-hms-rts
+```
+
+Then perform rollback:
+
+```bash
+helm -n services rollback cray-hms-rts <revision>
```

Comment:
Because we have to upgrade both cray-hms-rts and cray-hms-rts-snmp from the same helm chart, do we also need to add the following rollback in addition to what you have above?

helm -n services rollback cray-hms-rts-snmp

Reply:
Done!